src/compiler/builder.ts

```diff
@@ -372,6 +372,7 @@ namespace ts {
                 BuilderState.updateSignaturesFromCache(state, state.currentAffectedFilesSignatures!);
                 state.currentAffectedFilesSignatures!.clear();
                 BuilderState.updateExportedFilesMapFromCache(state, state.currentAffectedFilesExportedModulesMap);
+                state.currentAffectedFilesExportedModulesMap?.clear();

```

Comment:
I think this got missed when we first introduced currentAffectedFilesExportedModulesMap.. Look how we clear currentAffectedFilesSignatures just above..